src/aks-preview/azcli-aks-live-test/ext_matrix_default.json

```diff
@@ -0,0 +1,39 @@
+{
+    "coverage": {
+        "test_aks_commands": [
+            "AzureKubernetesServiceScenarioTest"
+        ]
+    },
+    "exclude": {
+        "naf": [
```

Comment:
what's naf

Reply:
Short for need additional feature, all the test cases under this field need to enable container service features such as "AutoUpgradePreview". By default, these test cases won't be tested.

Reply:
then use "needAdditionalFeature" instead?

Reply:
sure

Reply:
fixed in commit [47bb7f3](https://github.com/Azure/azure-cli-extensions/pull/3292/commits/47bb7f3b0ff9ae58f87bebc472f84f2366a19207).